sentry.go

```diff
@@ -298,7 +298,7 @@ func (hook *SentryHook) convertStackTrace(st errors.StackTrace) *raven.Stacktrac
 		pc := uintptr(stFrames[i])
 		fn := runtime.FuncForPC(pc)
 		file, line := fn.FileLine(pc)
-		frame := raven.NewStacktraceFrame(pc, file, line, stConfig.Context, stConfig.InAppPrefixes)
+		frame := raven.NewStacktraceFrame(pc, fn.Name(), file, line, stConfig.Context, stConfig.InAppPrefixes)
```

Comment:
this fn call need to fix quickly ...

Reply:
@evalphobia

Reply:
So only this line is needed to follow changed parameters on `NewStacktraceFrame` ?

Sorry, I thought this PR was accidentally created by mistake.
Should I create new PR or  @shreyasj2006 would you fix this PR ?